pkg/crc/machine/start.go

```diff
@@ -520,7 +520,7 @@ func (client *client) validateStartConfig(startConfig types.StartConfig) error {
 	return nil
 }
 
-func createHost(machineConfig config.MachineConfig) error {
+func createHost(machineConfig config.MachineConfig, isOpenShift bool) error {
```

Comment:
Maybe this should be part of `MachineConfig`? Not fully sure what this type represents exactly :)

Reply:
@cfergeau I can pass the preset if that make it more clear but I don't think we should have it part of MachineConfig just for this one.